Formula/g/gnupg.rb

```diff
@@ -23,6 +23,7 @@ class Gnupg < Formula
   end
 
   depends_on "pkgconf" => :build
+  # Please don't change this to ntbtls. https://github.com/Homebrew/homebrew-core/pull/218008#pullrequestreview-2738730628
```

Comment:
Let's wait to reach a decision there. New info: ntbTLS is being used for the official windows builds of GnuPG since 2017